test/end-to-end/browser/helpers/complete-risk-assessment.js

```diff
@@ -121,9 +138,13 @@ export const fullAssessmentRecommendation = (config) => {
 };
 
 export const thenTheAssessmentIsCompleted = (config = defaultAssessmentConfig) => {
-  DashboardPage.waitForMainHeadingWithDataId('dashboard');
+  if (config.prodSmokeTest) {
+    browser.url('/dashboard?displayTestAssessments=true');
+  }
+
+  expect(DashboardPage.waitForMainHeadingWithDataId('dashboard')).to.contain('All assessments');
 
-  browser.waitForVisible(`[data-element-id="profile-row-${config.prisoner.nomisId}"]`, 5000);
+  browser.waitForVisible(`tr[data-element-id="profile-row-${config.prisoner.nomisId}"]`, 5000);
```

Comment:
should we externalise these 5000 values or create a constant in the module for it?